tests/lean/run/463.lean

```diff
@@ -4,4 +4,4 @@ structure C := (a b c : Nat)
 structure D := (toA : A) (c : Nat)
 
 def foo (s : C) : B := {s with} -- works in lean 4, works in lean 3
-def bar (s : D) : B := {s with} -- works in lean 4, fails in lean 3
+-- def bar (s : D) : B := {s with} -- works in lean 4, fails in lean 3
```

Comment:
Correct me if I'm wrong, you don't need to comment this out, right?

You can add a comment that this is not necessarily meant to succeed.

Reply:
Doesn’t it fail a test because this errors?